code/game/gamemodes/cult/ritual.dm

```diff
@@ -463,10 +463,40 @@ var/global/list/rune_list = list() // HOLY FUCK WHY ARE WE LOOPING THROUGH THE W
 
 		if(usr.get_active_hand() != src)
 			return
-		for (var/mob/V in viewers(src))
-			V.show_message("<span class='warning'>[user] slices open a finger and begins to chant and paint symbols on the floor.</span>", 1, "<span class='warning'>You hear chanting.</span>", 2)
-		to_chat(user, "<span class='warning'>You slice open one of your fingers and begin drawing a rune on the floor whilst chanting the ritual that binds your life essence with the dark arcane energies flowing through the surrounding world.</span>")
-		user.take_overall_damage((rand(9)+1)/10) // 0.1 to 1.0 damage
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if(H.species.anatomy_flags & NO_BLOOD) //No blood, going to have to improvise
+				if(H.bloody_hands && H.held_items.len) //Blood on hand to use, and hands on hand to use
+					user.visible_message("<span class='warning'>[user] starts to paint drawings on the floor with the blood on their hands, whilst chanting.</span>",\
+					"<span class='warning'>You use the blood smeared on your hands to begin drawing a rune on the floor whilst chanting the ritual that binds your life essence with the dark arcane energies flowing through the surrounding world.</span>",\
+					"<span class='warning'>You hear chanting.</span>")
+					H.bloody_hands = max(0, H.bloody_hands - 1)
+				else //We'll have to search around for blood
+					var/turf/T = get_turf(user)
+					var/found = 0
+					if(H.held_items.len)
```

Comment:
Why not just move all of this check to the top right under ishuman(user)?

Obviously if they don't have hands they can't draw.